### PR TITLE
[plugin.video.nbcsportsliveextra] 2020.12.7 - Broken 

### PR DESCRIPTION
**Pull request by eracknaphobia**
### Description

Helix support is dropped in favor of isengard and newer.

### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

File: plugin.video.nbcsnliveextra/addon.xml [hunk 1]
```diff
@@ -19,6 +19,7 @@
     <forum>http://forum.kodi.tv/showthread.php?tid=207159</forum>
     <website></website>
     <email></email>
+    <broken>This add-on is broken and will no longer be under development.</broken>
```
Comment (enen92) [inline plugin.video.nbcsnliveextra/addon.xml]: users in helix receiving the update will think the addon will be no longer developed, when in fact it's available from isengard. Can you change this to a more explicit message which suggests them to update their kodi installation?

Comment (eracknaphobia) [reply plugin.video.nbcsnliveextra/addon.xml]: Ah yes, good point.

**Comment by enen92** [comment]
ERROR: Schema validation failed for the following points: xbmc.addon.metadata , disclaimer needs a Lang attribute

**Comment by enen92** [comment]
@eracknaphobia please redo the PR and use your own fork instead of a new branch on the upstream/main repo.

**Pull request closed without merging**